src/main/java/com/google/codeu/data/Marker.java

```diff
@@ -0,0 +1,26 @@
+package com.google.codeu.data;
+
+public class Marker {
+
+  private double lat;
+  private double lng;
+  private String content;
+
+  public Marker(double lat, double lng, String content) {
+    this.lat = lat;
+    this.lng = lng;
+    this.content = content;
+  }
+
+  public double getLat() {
```

Comment:
Is the intention to make these attributes read-only?

Reply:
Yes, currently there is no strong reason for the object to be mutable so to it is good to make it immutable. Also, it is consistent with the design pattern of the current implementation of `Message` class.